src/probnum/filtsmooth/gaussian/_kalman.py

```diff
@@ -21,28 +21,8 @@
 ]
 
 
-class Kalman(_bayesfiltsmooth.BayesFiltSmooth):
-    """Gaussian filtering and smoothing, i.e. Kalman-like filters and smoothers.
-
-    Parameters
-    ----------
-    prior_process
-        Prior Gauss-Markov process.
-        Usually a :class:`MarkovProcess` with a
-        :class:`Normal` initial random variable,
-        and an :class:`LTISDE` transition or an :class:`IntegratorTransition`,
-        but :class:`LinearSDE`, :class:`ContinuousEKFComponent`,
-        or :class:`ContinuousUKFComponent` are also valid.
-        Describes a random process in :math:`K` dimensions.
-        If the transition is an integrator, `K=d*(nu+1)` for some d and nu.
-    """
-
-    def iterated_filtsmooth(
-        self,
-        regression_problem: problems.TimeSeriesRegressionProblem,
-        init_posterior: Optional[_kalmanposterior.SmoothingPosterior] = None,
-        stopcrit: Optional[FiltSmoothStoppingCriterion] = None,
-    ):
+class _KalmanBase(_bayesfiltsmooth.BayesFiltSmooth):
+    def iterated_filtsmooth(self, *args, **kwargs):
```

Comment:
This wrapper around the state generator works with args and kwargs for the following reason:

Soon, the regression problems might change types (discrete vs continuous), and the `filter()` and `iterated_filtsmooth_generator()` signature adapts accordingly. But this function here (and the `filtsmooth()` style methods below) remains the same. args and kwargs lead to a smaller, more meaningful diff in the future.